src/sources/journald.rs

```diff
@@ -192,41 +192,19 @@ where
     T: Sink<SinkItem = Record, SinkError = ()>,
 {
     pub fn run(mut self) {
+        self.seek_checkpoint();
+
         let timeout = time::Duration::from_millis(500); // arbitrary timeout
```

Comment:
The batch size is already exposed as `batch_size: Option<usize>`. It may make sense to expose this polling time as well, although the `file` source has a similar kind of polling (a backoff, actually) and does not expose any parameters for it.